go/hyperkit.go

```diff
@@ -467,10 +472,7 @@ func (h *HyperKit) buildArgs(cmdline string) {
 
 	for _, p := range h.Disks {
 		// Default the driver to virtio-blk
-		driver := "virtio-blk"
```

Comment:
This change also is completely unrelated to the change described in the commit message

Reply:
Well, it uses a function introduced in this commit.  I anticipated that someone might claim that a single-use function is useless.

Reply:
This can still be done in a separate commit. You could mention in the commit which introduces the function as a single use function that subsequent commits will use it too. It's just as this commit stands now it changes too many different unrelated things

Reply:
Ack.